.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+on: push
+jobs:
+  build:
+    runs-on: macos-latest
+    env:
+      LDFLAGS: -L/usr/local/opt/openblas/lib
+      CPPFLAGS: -I/usr/local/opt/openblas/include
+      MAKEFLAGS: "-j $(sysctl -n hw.logicalcpu)"
+    steps:
+    - uses: actions/checkout@v2
+    - run: brew upgrade
+    - run: brew install openblas swig boost python3 tbb nlopt cminpack ceres-solver bison flex hdf5 ipopt r
+    - run: cd $HOME/work/openturns && git clone https://github.com/yixuan/spectra.git spectra
+    - run: cd $HOME/work/openturns/spectra && git checkout v0.9.0
+    - run: cd $HOME/work/openturns/spectra && cmake -DBUILD_TESTS=TRUE -DCMAKE_INSTALL_PREFIX=$HOME/ .
+    - run: cd $HOME/work/openturns/spectra && make install ${MAKEFLAGS}
+    - run: cd $HOME/work/openturns && git clone https://github.com/jeromerobert/hmat-oss.git hmat-oss
+    - run: cd $HOME/work/openturns/hmat-oss/ && git checkout 1.6.1
+    - run: cd $HOME/work/openturns/hmat-oss/ && cmake -DBUILD_EXAMPLES=OFF -DCBLAS_INCLUDE_DIR=/usr/local/opt/openblas/include -DCMAKE_MACOSX_RPATH=ON .
+    - run: cd $HOME/work/openturns/hmat-oss/ && make install ${MAKEFLAGS}
+    - run: pip3 install matplotlib scipy chaospy pandas dill
+    - run: cmake -DR_EXECUTABLE="dummyR" -DPYTHON_EXECUTABLE=/usr/local/bin/python3 -DFLEX_EXECUTABLE=/usr/local/opt/flex/bin/flex -DBISON_EXECUTABLE=/usr/local/opt/bison/bin/bison -DSPECTRA_INCLUDE_DIR=/Users/runner/include .
+    - run: make ${MAKEFLAGS}
```

Comment:
no need to specify ${MAKEFLAGS} for make commands

Reply:
seem that `makeflags` is generating some troubles